### PR TITLE
Profiling support

### PR DESCRIPTION
**Pull request by radubontila**
Added the missing features of profiling from sentry.

### CODE REVIEW

**Comment by amartiniuc** [comment]
up

**Comment by ipiticas** [comment]
👍

File: sentry/templates/deployment-vroom.yaml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+{{ toYaml .Values.vroom.securityContext | indent 8 }}
+      {{- end }}
+      containers:
+      - name: {{ .Chart.Name }}-vroom
+{{- if .Values.vroom.args }}
+        args:
+{{ toYaml .Values.vroom.args | indent 10 }}
+{{- end }}
+        image: "{{ template "vroom.image" . }}"
+        imagePullPolicy: {{ default "IfNotPresent" .Values.images.sentry.pullPolicy }}
```
Comment (adonskoy) [inline sentry/templates/deployment-vroom.yaml]:
```suggestion
        imagePullPolicy: {{ default "IfNotPresent" .Values.images.vroom.pullPolicy }}
```

File: sentry/templates/deployment-vroom.yaml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+{{/*        readinessProbe:*/}}
+{{/*          failureThreshold: {{ .Values.vroom.probeFailureThreshold }}*/}}
+{{/*          httpGet:*/}}
+{{/*            path: {{ template "relay.healthCheck.readinessRequestPath" }}*/}}
+{{/*            port: {{ template "relay.port" }}*/}}
+{{/*            scheme: HTTP*/}}
+{{/*          initialDelaySeconds: {{ .Values.vroom.probeInitialDelaySeconds }}*/}}
+{{/*          periodSeconds: {{ .Values.vroom.probePeriodSeconds }}*/}}
+{{/*          successThreshold: {{ .Values.vroom.probeSuccessThreshold }}*/}}
+{{/*          timeoutSeconds: {{ .Values.vroom.probeTimeoutSeconds }}*/}}
```
Comment (adonskoy) [inline sentry/templates/deployment-vroom.yaml]: Maybe it can be removed? Or add probes that will work (if possible)

**Review by adonskoy — CHANGES_REQUESTED** [review]
Some suggestions for vroom deployment

File: sentry/templates/deployment-vroom.yaml (hunk 1 above)
Comment (radubontila) [reply sentry/templates/deployment-vroom.yaml]: modified, ty

File: sentry/templates/deployment-vroom.yaml (hunk 2 above)
Comment (radubontila) [reply sentry/templates/deployment-vroom.yaml]: removed commented lines, could not find any health check to use

Comment (adonskoy) [reply sentry/templates/deployment-vroom.yaml]: There is a health check on [`/health`](https://github.com/getsentry/vroom/blob/2ec56bbfbea46a0272413359e147c66116adce68/cmd/vroom/main.go#L148)

**Comment by Mokto** [comment]
I merged 20.1.0. Could you please rebase your PR ?

**Review by Mokto — APPROVED** [review]

**Comment by Mokto** [comment]
Thanks a lot!

**Pull request merged**